meta-genivi-dev/recipes-sota/aktualizr/aktualizr_git.bb

```diff
@@ -12,19 +12,27 @@ S = "${WORKDIR}/git"
 
 SRCREV = "a6136f091d58b1bb97ec0dd1215eb570fa853f30"
 
-FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
-
 SRC_URI = " \
           git://github.com/advancedtelematic/aktualizr;branch=fix/boost-yocto \
+          file://aktualizr.service \
           "
 
+SYSTEMD_PACKAGES = "${PN}"
+SYSTEMD_SERVICE_${PN} = "aktualizr.service"
+
 DEPENDS = "boost curl openssl jansson dbus"
 RDEPENDS_aktualizr = "dbus-lib"
 
 EXTRA_OECMAKE = "-DWARNING_AS_ERROR=OFF -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTS=OFF -DBUILD_GENIVI=ON"
 
+do_install() {
+  install -d ${D}${systemd_unitdir}/system
```

Comment:
1. ${systemd_unitdir}/system -> ${systemd_system_unitdir}
2. I recommend that you use 4 spaces